### ISSUE TITLE
Hyperparameters when training

### ISSUE THREAD

**Issue by gondimjoaom** [vita-epfl/openpifpaf#288]
Hi, could you please recommend a good reference for studying the hyperparameters present on openpifpaf training?

I'm having these loss graphics after doing small changes, but I would like to decrease a little more my loss values and I believe this can be done be increasing the number of annotations, but also testing hyperparameters changing.

![DeepinScreenshot_select-area_20201008161825](https://user-images.githubusercontent.com/29072586/95511672-296dcd80-098e-11eb-9443-9ad26cba7e4e.png)

**Comment by svenkreiss** [comment]
Hello @gondimjoaom , at the moment, we don't have any further publications apart from the original paper and the Guide: https://vita-epfl.github.io/openpifpaf/dev/train.html 
The guide contains reference values for hyperparameters used to train the pretrained models in the example training commands. I am not aware of anyone who has tried to interface with fancier Bayesian hyperparameter optimization tools.

**Comment by gondimjoaom** [comment]
Got it! Thanks again for the enlightenment
